lib/td/command/job.rb

```diff
@@ -493,7 +496,7 @@ def render_result(job, limit, format=nil, render_opts={})
       job.result_each_with_compr_size {|row, compr_size|
         indicator.update(compr_size)
         rows << row.map {|v|
-          dump_column_safe_utf8(v)
+          dump_column_safe_utf8(v, render_opts[:null_expr])
```

Comment:
How about doing "Extract Local Variable" refactoring so that 3 codes looks smarter?
```
null_expr = render_opts[:null_expr]
...
      dump_column_safe_utf8(v,null_expr)
```